src/object/rock.cpp

```diff
@@ -151,11 +151,7 @@ Rock::collision(GameObject& other, const CollisionHit& hit)
 
   auto crusher = dynamic_cast<Crusher*> (&other);
   if (crusher) {
-    auto state = crusher->get_state();
-    if(state == Crusher::CrusherState::RECOVERING ||
-       state == Crusher::CrusherState::IDLE) {
-        return ABORT_MOVE;
-       }
+    return FORCE_MOVE;
```

Comment:
This if statement probably becomes useless. Idk. You'd have to test

Reply:
It doesn't.  It helps with making sure the crusher doesn't push the rock into the ground.  (I think.)